logos/src/logos/main.py

```
@@ -339,90 +520,288 @@ async def generalstats(data: LogosKeyModel):
+                        if first_chunk is None:
+                            first_chunk = blob
+                        if "choices" in blob and blob["choices"]:
+                            delta = blob["choices"][0].get("delta", {})
+                            content = delta.get("content", "")
+                            if content:
+                                full_text += content
+                    except json.JSONDecodeError:
+                        pass
+        except Exception as e:
+            error_message = str(e)
+            raise e
```

Ruff (0.14.8)

Use `raise` without specifying exception name

Remove exception name

(TRY201)